src/js/frontend/src/components/login/Login.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { GoogleLogin } from "@react-oauth/google";
+import * as Yup from "yup";
+import { Field, Form, Formik } from "formik";
+
+import { signInWithGoogleService } from "../../services/login/signInWithGoogleService";
+import { loginService } from "../../services/login/loginService";
+
+const Login = () => {
+  const [state, setState] = React.useState({
+    loginFailed: false,
+  });
+
+  const validationSchema = Yup.object().shape({
+    userName: Yup.string().required("Username Required"),
```

Comment:
'username' is fine, no need for 'userName'

Reply:
✅